### PR TITLE
 Added code of daily Coding Exercise-16

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/PrintTwoDimensionalArray.java [hunk 1]
```diff
@@ -13,9 +13,9 @@ public static void print2D(int mat[][]) {
...
     public static void main(String args[])
             throws IOException {
-        int matrix[][] = {{1, 2, 3, 4},
+        int mat[][] = {{1, 2, 3, 4},
                 {5, 6, 7, 8},
                 {9, 10, 11, 12}};
-        print2D(matrix);
+        print2D(mat);
     }
-}
```
Comment (AmrutaDhage20) [inline First/dailyExercises/PrintTwoDimensionalArray.java]: Remove the extra space.

File: First/dailyExercises/PrintTwoDimensionalArray.java [hunk 2]
```diff
@@ -13,9 +13,9 @@ public static void print2D(int mat[][]) {
 
     public static void main(String args[])
             throws IOException {
-        int matrix[][] = {{1, 2, 3, 4},
+        int mat[][] = {{1, 2, 3, 4},
                 {5, 6, 7, 8},
                 {9, 10, 11, 12}};
-        print2D(matrix);
+        print2D(mat);
```
Comment (AmrutaDhage20) [inline First/dailyExercises/PrintTwoDimensionalArray.java]: Give the proper variable name for(mat).

Comment (anushreeSonak) [reply First/dailyExercises/PrintTwoDimensionalArray.java]: Done !!

Comment (anushreeSonak) [reply First/dailyExercises/PrintTwoDimensionalArray.java]: Done !

**Review by AmrutaDhage20 — APPROVED** [review]

**Review by Vrushalee1234 — APPROVED** [review]

**Pull request merged**